webapp/i18n/en.json

```diff
@@ -818,6 +818,9 @@
   "admin.sql.keyTitle": "At Rest Encrypt Key:",
   "admin.sql.maxConnectionsDescription": "Maximum number of idle connections held open to the database.",
   "admin.sql.maxConnectionsExample": "E.g.: \"10\"",
+  "admin.sql.connectionTimeoutTitle": "Connection Timeout:",
+  "admin.sql.connectionTimeoutDescription": "The number of seconds a query can be executing for before being cancelled.",
```

Comment:
@JeffSchering Maybe have your help review? This setting will appear in the System Console > Database settings

I think there's an opportunity to specify what happens if/when an IT admin changes this value.